src/formatter-settings/FormatterConstants.ts

```diff
@@ -5,10 +5,17 @@ import { Category, ExampleKind, JavaFormatterSetting, ValueKind } from "./types"
 
 export namespace JavaConstants {
     export const JAVA_CORE_FORMATTER_ID = "org.eclipse.jdt.core.formatter";
+    export const CURRENT_FORMATTER_SETTINGS_VERSION = "21";
```

Comment:
Do we have better way to handle the version? If a new format version comes, you need to release a new version to handle with that.

Reply:
Here we can mark it as an engineering debt to find a proper solution later.